tensorflow/python/ops/image_ops_impl.py

```diff
@@ -323,6 +323,23 @@ def fix_image_flip_shape(image, result):
   return result
 
 
+@tf_export('image.random_flip')
+def random_flip(image, flip_index, seed=None):
```

Comment:
Can this signature take a generator instead, in conformance with the new rng stuff, instead of taking a seed?

Reply:
Thank you for the feedback, @alextp. So you are suggesting that I could use partial from functool
to let the signature directly take a generator like random_ops.random_uniform instead of seed, is that what you mean?

Reply:
I mean a tf.random.experimental.Generator https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/random/experimental/Generator

Reply:
Thanks for the feedback, I am currently at a conference and will work on it next week.